users/README.md

```
@@ -585,21 +587,44 @@ Creating a User Intro in DB if not available
 
 ## **PATCH /users/:userId**
 
-Allow SuperUser to approve the profilediff and Authenticated user to update their profile data.
+Allow SuperUser to approve the profilediff and any Authenticated user to update their own profile data.
 
 - **Params**
   - Optional: `userId=<userId>` 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)